src/main/java/teammates/logic/core/FeedbackResponsesLogic.java

```diff
@@ -201,7 +201,9 @@ public boolean isNameVisibleToUser(
         // Early return if user is giver
         if (question.giverType == FeedbackParticipantType.TEAMS) {
             // if response is given by team, then anyone in the team can see the response
-            if (roster.isStudentsInSameTeam(response.giver, userEmail)) {
+            // The second check is used to accommodate legacy data where team giver is a student email
```

Comment:
You meant the first check? The second check is the pre-existing one

Reply:
Yea, the pre-existing one is the one that's catering to legacy data I think. That's why @xpdavid suggested going ahead with this PR even though the session logic refactor meant that the original issue is no longer relevant

Reply:
I see, got it